src/Dapr.Actors/Runtime/ActorRuntime.cs

```diff
@@ -55,7 +55,7 @@ internal ActorRuntime(ActorRuntimeOptions options, ILoggerFactory loggerFactory,
         /// </summary>
         public IReadOnlyList<ActorRegistration> RegisteredActors => this.options.Actors;
 
-        internal static IDaprInteractor DaprInteractor => new DaprHttpInteractor();
+        internal static IDaprInteractor DaprInteractor => new DaprHttpInteractor(new System.Net.Http.HttpClientHandler());
```

Comment:
this instance should be non-static, and you can use `ActorRuntimeOptions` to initialize it. Remember, only the `HttpClientHandler` needs to be long-lived, these objects are cheap.